src/form-builder/DateField.tsx

```diff
@@ -29,6 +29,11 @@ const DateField = (props: DateProps): JSX.Element => {
     helpers.setValue((dateValue?.target as HTMLInputElement).value);
   };
 
+  const onInputChange = (e: Event) => {
```

Comment:
Any reason we use `onInputChange` here instead of `onChange` like the other components?

Reply:
`OnChange` already exist so we use `OnInputChange`